src/utils.js

```diff
@@ -21,11 +21,19 @@ let sendBeaconStatus = true;
 export default class Utils {
     constructor(targetWindow) {
         const timestamp = (+new Date()).toString(16);
-        const u32a = new Uint32Array(3);
         let result = '';
-        self.crypto.getRandomValues(u32a);
-        for (const num of u32a) {
-            result += num.toString(32)
+        if (self.crypto.getRandomValues) {
```

Comment:
On IE, `self.crypto` might be undefined. So this line will raise Uncaught TypeError.
It needs to be checked the existence of `crypto` object.

Reply:
fixed! thank you!